docs/features.md

```diff
@@ -389,7 +398,8 @@ Code action kind: `refactor.rewrite`
 Rewrites record selectors to use overloaded dot syntax
 
 ![Explicit Wildcard Demo](../plugins/hls-overloaded-record-dot-plugin/example.gif)
-## Missing features
+
+### Missing features
```

Comment:
I think this should stay `##`, it isn't a subsection of the plugin above.

Reply:
Yeah agree

Reply:
sorry for the mistake, I'll fix it in #3940